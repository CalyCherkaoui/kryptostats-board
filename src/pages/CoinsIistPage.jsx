import React from 'react';
import CoinsList from '../components/CoinsList';

function CoinsIistPage() {
  return (
    <div>
      <CoinsList />
    </div>
  );
}

export default CoinsIistPage;
