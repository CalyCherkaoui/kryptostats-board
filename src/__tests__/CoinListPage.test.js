import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../redux/store';
import CoinsIistPage from '../pages/CoinsIistPage';

describe(
  'Renders correctly Coin Detail Page container',
  () => {
    it(
      'Matches Coin Detail Page snapshot',
      () => {
        const rendered = renderer.create(
          <Provider store={store}>
            <Router>
              <CoinsIistPage />
            </Router>
          </Provider>,
        );
        expect(rendered).toMatchSnapshot();
      },
    );
  },
);
