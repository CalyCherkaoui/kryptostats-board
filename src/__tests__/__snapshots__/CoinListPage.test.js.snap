// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly Coin Detail Page container Matches Coin Detail Page snapshot 1`] = `
<div>
  <div
    className="coins_list_container"
  >
    <div
      className="coin_list_filters flex_row flex_start"
    >
      <div
        className="filter_wrapper flex_row"
      >
        <div
          className="filter_legend spaced_typography"
        >
          Convert prices to:
        </div>
        <select
          className="filter_selector"
          id="local_currency"
          name="local_currency"
          onChange={[Function]}
        >
          <option
            className="filter_option"
            value="usd"
          >
            USD
          </option>
          <option
            className="filter_option"
            value="eur"
          >
            EUR
          </option>
          <option
            className="filter_option"
            value="jpy"
          >
            JPY
          </option>
          <option
            className="filter_option"
            value="cad"
          >
            CAD
          </option>
          <option
            className="filter_option"
            value="btc"
          >
            BTC
          </option>
          <option
            className="filter_option"
            value="eth"
          >
            ETH
          </option>
        </select>
      </div>
    </div>
    <div
      className="coins_list"
    >
      <p
        className="nodata"
      >
        NO Data Available...
      </p>
    </div>
    <div
      className="pagination_wrapper"
    />
  </div>
</div>
`;
